Gemfile.lock

```diff
@@ -928,12 +926,12 @@ DEPENDENCIES
   aws-sdk-s3 (~> 1)
   baby_squeel (~> 1.4.3)
   bcrypt (~> 3.1.7)
-  bootsnap (~> 1.4)
+  bootsnap (~> 1.16)
   braintree (~> 2.93)
-  browser (~> 5.0.0)
+  browser
   bugsnag (~> 6.11)
   bullet (~> 6.1.5)
-  cancancan (~> 2.3.0)
+  cancancan (~> 3.0.0)
```

Comment:
this seems like  big leap, is it necessary to do together with all other rails things?

Reply:
I see this was required because of `Add support for Rails 6.0.`

Going through the migration guide
https://github.com/CanCanCommunity/cancancan/blob/develop/docs/migrating.md#from-2x-to-3x

Reply:
See #3481

Reply:
I merged master which already contains this change. No N+1 errors were detected in tests.